gcloud/connection.py

```diff
@@ -94,6 +93,25 @@ def http(self):
                 self._http = self._credentials.authorize(self._http)
         return self._http
 
+    @staticmethod
+    def _scoped_credentials(credentials, scope):
```

Comment:
Sure. Didn't put a lot of thought into it since it is private and lives on a base class.